ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -452,6 +452,35 @@ tests:
         requests:
           cpu: 400m
           memory: 1Gi
+- as: cluster-api-provider-nutanix
+  cron: 10 13 * * Mon,Thu
+  steps:
+    test:
+    - as: cluster-api-provider-nutanix
+      commands: |
+        rebasebot --source https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix:main \
+                  --dest openshift/cluster-api-provider-nutanix:main \
+                  --rebase openshift-cloud-team/cluster-api-provider-nutanix:rebase-bot-main \
+                  --update-go-modules \
+                  --slack-webhook /secrets/rebasebot/slack-webhook \
+                  --github-app-key /secrets/rebasebot/cloud-team-openshift-bot-key \
+                  --github-cloner-key /secrets/rebasebot/cloud-team-bot-key \
+                  --git-username cloud-team-rebase-bot \
+                  --git-email cloud-team-rebase-bot@redhat.com \
+                  --bot-emails cloud-team-rebase-bot@redhat.com openshift-bot@redhat.com openshift-ci-robot@redhat.com \
+                  --tag-policy=strict
+      credentials:
+      - collection: ""
+        mount_path: /secrets/rebasebot
+        name: rebase-bot-creds
+        namespace: test-credentials
+      from: rebasebot
+      resources:
+        limits:
+          memory: 6Gi
+        requests:
+          cpu: 400m
+          memory: 1Gi
```

Comment:
I think you copied the config from one of the cloud providers.
You should instead copy one from the `cluster-api-provider-*` from above. And put this near those up there.